code/modules/vehicles/mecha/combat/durand.dm

```diff
@@ -50,7 +50,7 @@
 
 /obj/vehicle/sealed/mecha/durand/process()
 	. = ..()
-	if(defense_mode && !use_power(100)) //Defence mode can only be on with a occupant so we check if one of them can toggle it and toggle
+	if(defense_mode && !use_energy(100)) //Defence mode can only be on with a occupant so we check if one of them can toggle it and toggle
```

Comment:
```suggestion
	if(defense_mode && !use_energy(0.1 KILO JOULES)) //Defence mode can only be on with a occupant so we check if one of them can toggle it and toggle
```

Reply:
I'll also scale this since this uses the cell.